core/domain/story_services_test.py

```diff
@@ -502,6 +514,21 @@ def test_get_completed_node_ids(self):
                 self.owner_id, self.STORY_1_ID),
             [self.NODE_ID_1, self.NODE_ID_2, self.NODE_ID_3])
 
+    def test_get_latest_completed_node_ids(self):
```

Comment:
Test other cases as well -- record completion in different orders, record completion multiple times, etc. That way this remains robust even if UI decisions subsequently change.

Reply:
Done.